apps/site/assets/ts/stop/components/Departures.tsx

```diff
@@ -64,20 +84,45 @@ const Departures = ({
             modeButtonsToShow={availableModes(routes)}
           />
         )}
-
-        {filteredRoutes.map(routeWithDirections => (
-          <RouteCard
-            key={routeWithDirections.route.id}
-            route={routeWithDirections.route}
-            directions={routeWithDirections.directions}
-            stop={stop}
-            alerts={
-              routesAndAlerts
-                ? routesAndAlerts[routeWithDirections.route.id]
-                : []
-            }
-          />
-        ))}
+        <div className="hidden-sm-down mt-8">
+          {filteredRoutes.map(routeWithDirections => (
+            <RouteCard
+              key={routeWithDirections.route.id}
+              route={routeWithDirections.route}
+              directions={routeWithDirections.directions}
+              stop={stop}
+              alerts={
+                routesAndAlerts
+                  ? routesAndAlerts[routeWithDirections.route.id]
+                  : []
+              }
+            />
+          ))}
+        </div>
+        <div className="hidden-md-up mt-8">
+          {/* This should only show on small screens */}
+          {shortFilteredRoutes.map(routeWithDirections => (
+            <RouteCard
+              key={routeWithDirections.route.id}
+              route={routeWithDirections.route}
+              directions={routeWithDirections.directions}
+              stop={stop}
+              alerts={
+                routesAndAlerts
+                  ? routesAndAlerts[routeWithDirections.route.id]
+                  : []
+              }
+            />
+          ))}
+          {!showMore && filteredRoutes.length > 2 && (
```

Comment:
Maybe check for whether a mode is selected here?
I just ran into the edge case where, on South Station, I select "Rail", it gives me all the rail, BUT still shows the "show 6 more routes" button. 
<img width="356" alt="image" src="https://user-images.githubusercontent.com/2136286/214684398-fac0f933-51a3-4814-9599-fb3271dfe12d.png">